modules/provision/main.tf

```diff
@@ -36,10 +36,39 @@ resource "aws_iam_policy" "vespa_cloud_provision_policy" {
   }
 }
 
-# TODO: Remove this when we no longer depend on parameter store
 resource "aws_iam_policy" "vespa_cloud_host_policy" {
   name   = "vespa-cloud-host-policy"
-  policy = file("${path.module}/host-policy.json")
+  policy = jsonencode({
```

Comment:
I like it when these are `aws_iam_policy_document`, but not critical to fix in this PR.

Reply:
Unless it's very big, like the `provisioner-policy.json`, I prefer them in-file because then you can avoid JSON limitations like trailing commas on last element and comments

Reply:
I totally agree. Was not objecting to the policy being in the Terraform file. Just that we could have had the policy document as a separate resource.